workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] At the moment, this linter check is not running because it is not inside a `.github` folder. To fix this please make the structure should be `.github/workflows/linters.yml`